src/main/java/sirius/web/templates/DefaultGlobalContextExtender.java

```diff
@@ -61,6 +61,7 @@ public void collectTemplate(Collector parameterCollector) {
         parameterCollector.collect("lang", NLS.getCurrentLang());
         parameterCollector.collect("contentHelper", ContentHelper.INSTANCE);
         parameterCollector.collect("wondergemRoot", wondergemRoot, String.class);
+        parameterCollector.collect("tychoRoot", wondergemRoot, String.class);
```

Comment:
Should this be the same value as wondergem root?

Reply:
yop for backwards compatibility